### DISCUSSION TITLE
Can I play songs from spotify and youtube?

### DISCUSSION THREAD

**Discussion by ultra-black** [SudhanPlayz/Discord-MusicBot#729] [Q&A] [answered]
If I did lava link and spotify.
Will it play both Youtube and Spotify?

**Comment by JotaroKujo0525** [comment]
Short question, yes.

Also, the lavalink node is converting those Spotify links and searching YouTube for the exact result.

**Comment by DarrenOfficial** [comment, marked as answer]
Can it play from YouTube? Absolutely yes.
Can it play from Spotify? No. lavalink & the bot doesn't support playback from Spotify, what the bot is doing is converting Spotify links to YouTube links and play it. There is no way to directly play from Spotify due to its strict DRM.

You can see all of the exact format that lavalink support here https://github.com/sedmelluq/lavaplayer#supported-formats
